worker/sandbox/settings.go

```diff
@@ -0,0 +1,24 @@
+package sandbox
+
+type Settings struct {
+	LogSandbox bool
+	IgnoreWarning bool
+}
+
+var DefaultSettings = Settings{LogSandbox: true, IgnoreWarning: false}
+
+type Option func(Settings) Settings
+
+func IgnoreWarnings(ignore bool) Option {
+	return func(o Settings) Settings {
+		o.IgnoreWarning = ignore
+		return o
+	}
+}
+
+func EnableSandboxLogs(enable bool) Option {
```

Comment:
```suggestion
// EnableSandboxLogs sets whether sandbox logs should be printed.
func EnableSandboxLogs(enable bool) Option {
```
Also maybe a more idiomatic name should be `WithSandboxLogs`.